pkg/bench/ring_check.go

```diff
@@ -55,7 +55,7 @@ func NewRingChecker(id string, instanceName string, cfg RingCheckConfig, workloa
 		workload: workload,
 	}
 	reg := prometheus.DefaultRegisterer
-	cfg.MemberlistKV.MetricsRegisterer = reg
+	//cfg.MemberlistKV.MetricsRegisterer = reg
```

Comment:
Curious why this was commented out?

Reply:
This was removed from dskit recently
https://github.com/grafana/dskit/pull/327/files